lib/performify/base.rb

```diff
@@ -70,7 +70,9 @@ def fail?
       @result.blank?
     end
 
-    private def prepare_instance
+    private
```

Comment:
Please, do not make code style changes: `private` is supposed to be attached to the method here, it was not a mistake.